backend/capellacollab/settings/integrations/purevariants/models.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import logging
+import typing as t
+
+import pydantic
+import requests
+from pydantic import BaseModel
+from requests import RequestException
+from sqlalchemy import Column, String
+
+from capellacollab.core.database import Base
+
+log = logging.getLogger(__name__)
+
+
+def validate_license_url(value: t.Optional[str]):
+    if value:
+        try:
+            requests.Request("GET", value).prepare()
+        except RequestException:
+            log.info("Floating license validation failed", exc_info=True)
```

Comment:
We do not need a stack trace here.

Reply:
It's nice for finding the root cause of a failed validation, e.g. we want to add a license server URL, but the validation fails. With the stacktrace in the logs, we can identify the reason more easily.